ocs_ci/ocs/perftests.py

```diff
@@ -505,3 +509,140 @@ def get_time():
 
         """
         return time.strftime("%Y-%m-%dT%H:%M:%SGMT", time.gmtime())
+
+    def check_tests_results(self):
+        """
+        Check that all sub-tests (test multiplication by parameters) finished and
+        pushed the data to the ElastiSearch server.
+        It also generate the es link to push into the performance dashboard.
+        """
+
+        es_links = []
+        try:
+            input_file = open(self.results_file, "r")
+            data = input_file.read().split("\n")
+            data.pop()  # remove the last empty element
+            input_file.close()
+            if len(data) != self.number_of_tests:
+                log.error("Not all tests finished")
+                raise exceptions.BenchmarkTestFailed()
+            else:
+                log.info("All test finished OK, and the results can be found at :")
+                for res in data:
+                    log.info(res)
+                    es_links.append(res)
+        except OSError as err:
+            log.error(f"OS error: {err}")
+            raise err
+
+        self.es_link = ",".join(es_links)
+
+    def push_to_dashboard(self, test_name):
+        """
+        Pushing the test results into the performance dashboard, if exist
+
+        Args:
+            test_name (str): the test name as defined in the performance dashboard
+
+        :return:
+        """
+
+        try:
+            db = PerfDash()
+        except MissingRequiredConfigKeyError as ex:
+            log.error(
+                f"Results can not push to performance dashboard, no connection [{ex}]"
```

Comment:
Spelling fix - "Results cannot be pushed to the performance dashboard"

Reply:
fixed.